packages/loot-design/src/components/budget/index.js

```diff
@@ -729,7 +730,11 @@ function ExpenseGroup({
   );
 }
 
-function ExpenseCategory({
+const ExpenseCategory = connect(state => ({
+  isNewAutocompleteEnabled: state.prefs.local['flags.newAutocomplete'],
+}))(ExpenseCategoryInternal);
```

Comment:
A hack.. but the alternative was to drill the prop like 7 layers which is even more nasty. So IMO this is fine as we plan to clean this up in the near term future.